app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,5 @@
 <resources>
-    <string name="app_name">Nothing3AppKt</string>
+    <string name="app_name">EmokisApp</string>
```

Comment:
@lalongooo App suffix in name is kind of redundant here, actually Emokis is the real name 😉

`<string name="app_name">Emokis</string>`